packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1270,6 +1279,36 @@ describe('ReactDOMInput', () => {
       expect(node.getAttribute('value')).toBe('2');
     });
 
+    it('initially sets the value attribute on mount', () => {
```

Comment:
Can you explain why these tests were added, and why they are important?

Reply:
Good catch. These aren't necessary, but I needed them mid-way through making the change. I'll remove these.

I added these tests when I removed the `value` mutation method and [removed `value` from the property object in `getHostProps`](https://github.com/facebook/react/pull/11534/files#diff-2364bccf91025269af6a1cc9483ffdb3R67). Without the addition of [assigning `value` in the post mount hook](https://github.com/facebook/react/pull/11534/files#diff-2364bccf91025269af6a1cc9483ffdb3R223), inputs never received an initial value.

But it looks like these aren't necessary any more. I've also since removed specific checks for `reset` and `submit` input types too. Checking again, half the ReactDOMInput test suite fails when the value assignment code in `postMountWrapper` goes away.

Reply:
Removed in fde83b2